README.md

```
@@ -7,7 +7,7 @@
     <img src="https://npmx.dev/api/registry/badge/version/element-plus" />
   </a>
   <a href="https://github.com/element-plus/element-plus">
-    <img src="https://img.shields.io/badge/node-%20%3E%3D%2020-47c219" />
+    <img src="https://img.shields.io/badge/node-%20%3E%3D%2022-47c219" />
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)